bolt/src/main/java/com/slack/api/bolt/App.java

```diff
@@ -1346,10 +1370,15 @@ protected Response runHandler(Request slackRequest) throws IOException, SlackApi
                         return Response.ok();
                     }
                     EventRequest request = new EventRequest(stepRequest.getRequestBodyAsString(), stepRequest.getHeaders());
-                    BoltEventHandler<Event> handler = eventHandlers.get(request.getEventTypeAndSubtype());
-                    if (handler != null) {
-                        EventsApiPayload<Event> payload = buildEventPayload(request);
-                        return handler.apply(payload, request.getContext());
+                    List<BoltEventHandler<Event>> handlers = eventHandlers.get(request.getEventTypeAndSubtype());
```

Comment:
This duplicated code is for Steps from apps (the legacy workflow enhancement). The code can be improved not to repeat the same lines of code but not this time.